dbt/models/reporting/reporting.vw_pin_township_class.sql

```diff
@@ -9,6 +9,59 @@ WITH correct_class AS (
         deactivat,
         cur
     FROM {{ source('iasworld', 'pardat') }}
+),
+
+/* This CTE creates our AHSAP indicator. It's a function of multiple columns
```

Comment:
suggestion (non-blocking): Maybe add a small bit of context here for those unfamiliar with the AHSAP program and/or why the indicator is so complicated.